modules/compute/function-app/main.bicep

```diff
@@ -0,0 +1,299 @@
+/*param section*/
+@description('Required. Name for the Azure Function App.')
+param name string
+
+@description('Required. Location for all resources.')
+param location string
+
+@description('Optional. Tags for all resources within Azure Function App module.')
+param tags object = {}
+
+@description('Required. Defines the name, tier, size, family and capacity of the app service plan.')
+param sku object
```

Comment:
```suggestion
param sku object = {
  name: 'Y1'
  tier: 'Dynamic'
  size: 'Y1'
  family: 'Y'
  capacity: 0
}
```

Reply:
We can include a default value here. This helps to show users how the object should be created.